@here/harp-mapview/lib/MapView.ts

```diff
@@ -2833,10 +2834,10 @@ export class MapView extends THREE.EventDispatcher {
         }
 
         this.m_textElementsRenderer.placeText(
-            this.checkIfTextElementsChanged(),
-            this.checkIfTilesChanged(),
+            this.m_visibleTiles.dataSourceTileList,
+            this.projection,
             time,
-            this.m_frameNumber
+            this.checkIfTilesChanged()
```

Comment:
Why not delay evaluating this until you need it, I see that the placeText can early exit before needing to compute this value.

Reply:
done